packages/rx_bloc_cli/example/README.md

```diff
@@ -13,15 +13,15 @@
 9. [Design system](#design-system)
 10. [Golden tests](#golden-tests)
 11. [Server](#server)
-12. [Push notifications](#push-notifications)
-13. [Social Logins](#social-logins-library)
-14. [Dev Menu](#dev-menu)
-15. [Patrol integration tests](#patrol-integration-tests)
-16. [Realtime communication](#realtime-communication)
-17. [Feature OTP](#feature-otp)
-18. [CI/CD](#cicd)
-19. [Feature Pin Code](#feature-pin-code)
-20. [Auth Matrix](#auth-matrix)
+12. [Push notifications](#push-notifications){{#enable_social_logins}}
```

Comment:
Since this is not a brick, but a generated file, which is visible here https://pub.dev/packages/rx_bloc_cli/example it should not have any placeholders

Reply:
Yes, I am aware of the issue which resurfaced. Currently looking into why it is happening.